jobs/build/custom/Jenkinsfile

```diff
@@ -71,6 +71,12 @@ node {
                         $class: 'hudson.model.StringParameterDefinition',
                         defaultValue: "",
                     ],
+                    [
+                        name: 'ADVISORY_ID',
```

Comment:
If this is only for image advisory ID, then it should be named `IMAGE_ADVISORY` or something.